src/Compilers/CSharp/Portable/CodeGen/EmitAddress.cs

```diff
@@ -491,16 +520,16 @@ private void EmitParameterAddress(BoundParameter parameter)
         }
 
         /// <summary>
-        /// Emits receiver in a form that allows member accesses ( O or &amp; ). For verifiably
-        /// reference types it is the actual reference. For generic types it is a address of the
-        /// receiver with readonly intent. For the value types it is an address of the receiver.
+        /// Emits receiver in a form that allows member accesses ( O or &amp; ). 
+        /// For verifiably reference types it is the actual reference. 
```

Comment:
Here the term is used as "reference type for the purpose of verification", which for example does not include "T as class".
Will change it to "verifier-reference"

---
In reply to: [108191572](https://github.com/dotnet/roslyn/pull/18175#discussion_r108191572) [](ancestors = 108191572)